src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java

```diff
@@ -90,6 +92,12 @@ public void audit() {
         response = ossIndexClient.requestComponentReports(packageUrls);
       }
 
+      Set<String> vulnerabilityIds = extension.getExcludeVulnerabilityIds();
```

Comment:
Maybe `vulnerabilityIds` to `vulnerabilityIdsToExclude`?

Reply:
> Maybe `vulnerabilityIds` to `vulnerabilityIdsToExclude`?

👍 fc1fe2a5